io_context/CMakeLists.txt

```diff
@@ -52,12 +52,16 @@ if(BUILD_TESTING)
 
   ament_add_gtest(${TEST_UDP_DRIVER_EXE}
           test/${name}.cpp)
-  ament_target_dependencies(${TEST_UDP_DRIVER_EXE} std_msgs Boost)
+  ament_target_dependencies(${TEST_UDP_DRIVER_EXE} std_msgs "ASIO")
```

Comment:
@JWhitleyWork does the 'ament_auto' for testing find the std_msgs dependency automatically?